workflow/client_test.go

```diff
@@ -595,7 +595,7 @@ func TestCompleteCancelledActivity(t *testing.T) {
 		expectedActivity := &models.Activity{
 			ID:     swag.String(activityID),
 			Status: swag.String(models.ActivityStatusCancelled),
-			Error:  &models.ActivityError{Details: "some cancel details"},
+			Error:  &models.ActivityError{Reason: swag.String("some reason"), Details: "some cancel details"},
```

Comment:
Formatting?

Reply:
@pkrull What's wrong with the formatting?

Reply:
Ah, I personally only break initializers across lines if the line is too long.  1 or 2 fields, is okay.  More than that and probably want multiple lines.